sidebars.js

```diff
@@ -55,8 +55,10 @@ const sidebars = {
           collapsible: true,
           collapsed: true,
           items: [
+            'building-on-lisk/interacting-with-the-blockchain/web3',
             'building-on-lisk/interacting-with-the-blockchain/ethers',
             'building-on-lisk/interacting-with-the-blockchain/viem',
```

Comment:
Maybe we should put viem at first, as it as an integration for the OP stack
```suggestion
            'building-on-lisk/interacting-with-the-blockchain/viem',
            'building-on-lisk/interacting-with-the-blockchain/ethers',
            'building-on-lisk/interacting-with-the-blockchain/web3',
```